File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 3 - Python/q7.py
#Crie um algoritmo que receba um número e imprima uma mensagem se for múltiplo de 4 e 8 ou de 6 e 9.

x=int(input("Insira um número: "))
if x%4==0 and x%8==0:
    print(x, "é mútiplo de 4 e 8")
else:
    if x%6==0 and x%9==0:
        print(x, "é mútiplo de 6 e 9")
    

File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 3 - C/q7.c
//. Crie um algoritmo que receba um número e imprima uma mensagem se for múltiplo de 4 e 8 ou de 6 e 9.

#include<stdio.h>
int main(){
	int x;
	printf("Insira um valor: ");
	scanf("%d", &x);
	if(x%4==0 && x%8==0)  
		printf("Divisivel por 4 e 8\n");
	else if(x%6==0 && x%9==0)
		printf("Divisivel por 6 e 9\n");
}
